admin_panel/admin.py

```diff
@@ -0,0 +1,60 @@
+from admin_numeric_filter.admin import RangeNumericFilter
+from django.contrib import admin
+from django.contrib.admin import SimpleListFilter
+
+from .models import Product, ProductImage, Shop, Category
+
+
+# Register your models here.
+class ShopAdmin(admin.ModelAdmin):
+    search_fields = ('title',)
+    readonly_fields = ('thumbnail', 'id')
+
+
+class ProductImageInline(admin.TabularInline):
+    model = ProductImage
+    extra = 0
+    readonly_fields = ('thumbnail',)
+
+
+class ProductAdmin(admin.ModelAdmin):
+    search_fields = ("id", "title")
+    list_display = ('__str__', 'price', 'amount', 'main_image',)
+    inlines = (ProductImageInline,)
+    readonly_fields = ('id', 'main_image')
+    ordering = ('price', 'amount')
+    list_filter = (("price", RangeNumericFilter), "active")
+
+
+class ByProductListFilter(SimpleListFilter):
```

Comment:
Don't copy my code, please. This is even with my comments.

[Proof (my code)](https://github.com/Sariellee/django-test-task/blob/9b2bb1fb25d58243ae61c0624757ba62f1e1fe05/KE_test/admin/category.py#L6)

Reply:
@Sariellee don't worry, checking of such behaviour is also part of the test task